crates/devolutions-pedm-shell-ext/AppxManifest.xml

```diff
@@ -10,10 +10,10 @@
     xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"
     xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
     IgnorableNamespaces="uap uap2 uap3 rescap desktop desktop4 desktop5 uap10 com">
-    <Identity Name="DevolutionsAgent" ProcessorArchitecture="neutral" Publisher="CN=Devolutions"
-        Version="1.0.0.0" />
+    <Identity Name="DevolutionPEDMShellExtension" ProcessorArchitecture="neutral" Publisher="CN=Devolutions"
+        Version="2024.3.6.0" />
```

Comment:
isn't this updated in the CI, or is my memory incorrect and I never scripted that?

Reply:
I see the msix getting manipulated but nowhere is the version being set AFAICT. This tracks with the fact that the published package has a 1.0.0.0 version.

Reply:
my bad, I didn't recall we had a script for bumping the version numbers everywhere